### PR TITLE
Completed css-ecommerce-page exercise.

### PR DESCRIPTION
**Pull request by EuichangPark**
css-ecommerce-page

### CODE REVIEW

File: css-ecommerce-page/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      <div class="col-2">
+        <p>Tiffany <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Louis Vuitton <span>|One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Reformation <span> |Size 4</span></p>
+      </div>
+    </div>
```
Comment (TimelikeClosure) [inline css-ecommerce-page/index.html:50]: There should be more space between the separator and the size. Also, the separator should be taller relative to the text:

---

image

---

image

File: css-ecommerce-page/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
```
Comment (TimelikeClosure) [inline css-ecommerce-page/index.html:13]: Semantically, this should be a heading (`<h#>`), not a paragraph (`<p>`). Also, it needs a higher font-weight:

---

image

---

image

File: css-ecommerce-page/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p>Free people <span> |Size 6</span></p>
+      </div>
```
Comment (TimelikeClosure) [inline css-ecommerce-page/index.html:37]: The capitalization is off here:

---

image

File: css-ecommerce-page/index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      <div class="col-2">
+        <img src="images/key.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
```
Comment (TimelikeClosure) [inline css-ecommerce-page/index.html:33]: The different products should have some space between their images:

---

image

---

image

File: css-ecommerce-page/index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      <div class="col-2">
+        <p><span class="peach-background color-black">$476</span>&nbsp;<span><s>$2000 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$95</span>&nbsp;<span><s>$1200 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$20</span>&nbsp;<span><s>$198 retail</s></span></p>
+      </div>
+    </div>
```
Comment (TimelikeClosure) [inline css-ecommerce-page/index.html:67]: Content-wise, related information should be grouped together as much as possible. For example, a product's image, name, size, and pricing are all related, so their grouping should take priority over, for example, grouping all the product images together.

---

image

File: css-ecommerce-page/index.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$476</span>&nbsp;<span><s>$2000 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$95</span>&nbsp;<span><s>$1200 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$20</span>&nbsp;<span><s>$198 retail</s></span></p>
+      </div>
```
Comment (TimelikeClosure) [inline css-ecommerce-page/index.html:66]: The sale prices could use a little spacing around them within the background highlighting, and the background highlight color could be a little more subtle:

---

image

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**